bin/test

```diff
@@ -40,6 +40,7 @@ else
   # When not using modules, $build is setup as the GOPATH by compile w/$GO_SETUP_GOPATH_IN_IMAGE
   export GOPATH="${build}"
   PATH="${GOPATH}/bin:${GOROOT}/bin:${PATH}"
+  export GO111MODULE=off
```

Comment:
Guessing this was part of changes to make go1.13 the default, still needed if this PR isn't doing that?

Reply:
it's probably still wise. I can remove it though if the consensus is that I should.